src/renderer/components/VideoController.tsx

```diff
@@ -0,0 +1,41 @@
+import { styled, Box } from '@mui/material';
+import PlayArrowIcon from '@mui/icons-material/PlayArrow';
+import Forward10Icon from '@mui/icons-material/Forward10';
+import Replay10Icon from '@mui/icons-material/Replay10';
+import colors from '../colors';
+
+const VideoControllerBox = styled(Box)`
+  background: ${colors.grey[700]};
+  color: ${colors.grey[300]};
+  height: 62px;
+  width: 100vw;
+  margin: 0;
+  padding: 11px 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  column-gap: 24px;
+`;
+
+const VideoController = () => {
+  return (
+    <VideoControllerBox>
+      <div
+        style={{
+          backgroundColor: colors.grey[600],
+          fontWeight: 'bold',
```

Comment:
Could you change this to regular, size 28?

Reply:
Done and done, will push changes momentarily